titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/agent/replicator/GrpcAgentReplicatorEventStream.java

```diff
@@ -56,7 +59,10 @@ public GrpcAgentReplicatorEventStream(AgentManagementClient client,
                     metrics.event(titusRuntime.getClock().wallTime() - event.getLastUpdateTime());
                 })
                 .doOnUnsubscribe(metrics::disconnected)
-                .doOnError(metrics::disconnected)
+                .doOnError(error -> {
+                    logger.warn("Connection to the agent event stream terminated with an error: {}", error.getMessage(), error);
```

Comment:
nit: `logger.error`?

Reply:
Connection failure is most of the time non-critical issue (could be master failover, gateway termination, etc).